confluent-registry-schema/1.0/modules/ROOT/pages/confluent-schema-registry-connector-examples.adoc

```diff
@@ -0,0 +1,345 @@
+= Confluent Schema Registry 1.0 Examples - Mule 4
+
+This example shows how to publish and consume avro messages by integrating
+Confluent Schema Registry with Kafka Connector to reduce the size of the
+messages. The example contains two flows:
+
+. The first flow generates an avro message containing the schema and sends it
+to the Kafka topic.
+. The second flow reads the avro message from the Kafka topic and prints out the
+content of the deserialized avro message to the console.
+
+== Prerequisites
+
+You must have:
+
+* Java 8
+* Anypoint Studio 7.7.0 or later
+* Mule Runtime 4.2.x EE or later
+* DataWeave
+
+== Create the Producer Flow
+
+The Producer flow publishes a message to the configured Kafka topic. The
+following screenshot shows the Producer flow in Anypoint Studio:
+
+image::confluent-schema-registry-producer-flow.png[Confluent Schema Registry Connector Producer Flow in Studio]
+
+This flow involves configuring an HTTP *Listener* component, *Transform Message*
+component, *Logger* component, *Replace AVRO schema with id* operation, *Publish*
+operation, and a second *Transform Message* component.
+
+=== Configure HTTP Listener
+
+Configure *HTTP Listener* to initiate a Mule flow when a call is made to the `/`
+account path on localhost, port 8082.
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to
+that file from the connector configuration.
+
+If you don't know how to use a properties file,
+see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure *HTTP Listener*:
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for `http` and select the *Listener* operation:
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click the *Add* icon next to the *Connector configuration*
+field to add a global element.
+. Click *OK* to accept the defaults.
+. Set the *Path* field to `/`:
+
+=== Add the First Transform Message Component
+
+The first *Transform Message* component generates an avro message containing
+the schema:
+
+. In the *Mule Palette* view, search for *transform message*.
+. Drag the *Transform Message* component onto the canvas, next to the
+*Listener* component.
+. In the *Transform Message* configuration, overlay the brackets in the *Output*
+section with this XML:
++
+[source,xml,linenums]
+----
+%dw 2.0
+output application/avro schemaUrl='classpath://schema.json'
```

Comment:
shouldn't the content of the schema be specified somewhere? Because these schema is specific to the used example.

Reply:
Good idea. Just added in lines 81-85